log/uds.go

```diff
@@ -84,12 +86,14 @@ func (u *udsCore) Check(e zapcore.Entry, ce *zapcore.CheckedEntry) *zapcore.Chec
 
 // Sync implements zapcore.Core. It sends log messages with HTTP POST.
 func (u *udsCore) Sync() error {
+	u.mu.Lock()
```

Comment:
Nit: consider a private func “logsFromBuffer()” which will do locking in the beginning and unlocking at the end. Since it has functions that can panic defer may be appropriate in that function.

Reply:
Done.